docs/src/pages/[platform]/connected-components/in-app-messaging/examples/custom-components.react.mdx

```diff
@@ -0,0 +1,9 @@
+import { Tabs, TabItem, } from '@aws-amplify/ui-react';
+import { Example, ExampleCode } from '@/components/Example';
+
+<Example>
+  <ExampleCode>
+    ```js file=../../../../../../../examples/next/pages/ui/components/in-app-messaging/override-component/index.page.tsx
```

Comment:
Should this be tsx?

```suggestion
    ```tsx file=../../../../../../../examples/next/pages/ui/components/in-app-messaging/override-component/index.page.tsx
```

Reply:
Same question with other ```s below

Reply:
Since these are not specifically TS examples i opted for leaving them as just `js`